appium/.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.2
```

Comment:
You don't know what language versions the users are using! This is perfectly acceptable for applications to set the specific language version, but not for a library. Tying a library to a specific language version is not a good idea. 

To keep the usability of their libraries, often when releasing new library versions, the maintainer often states in the release log - "Tested against a range of language/dependencies versions".

I would:
 - add the `.ruby-version` file to .gitignore (because I want to use it internally, but I don't want to burden the other maintainers with a specific language version)
 - increase the gem version in https://github.com/ivarbanovv/screen-object/blob/master/appium/lib/screen-object/version.rb (I would follow semantic versioning)
 - test the library against several ruby versions (are at least with the modern ones -> 2.7, 3.0, 3.1)

Reply:
Thanks 🙇🏻 
`.ruby-version` file removed 🪓